baremetal/metal3data_manager.go

```diff
@@ -372,20 +386,24 @@ func (m *DataManager) getAddressesFromPool(ctx context.Context,
 		}
 
 		for _, network := range m3dt.Spec.NetworkData.Networks.IPv6 {
+			m.Log.Info("Fetch network data from IPPool for IPv6", "IPPool", network.IPAddressFromIPPool)
```

Comment:
Since you are adding IPv6 here, I am thinking would it be beneficial to follow the same trend in other messages as well to some extent if not in all?

Reply:
I'll update them :+1: